wiki/Beatmap/Circle_size/ja.md

```diff
@@ -0,0 +1,38 @@
+---
+tags:
+  - CS
+  - key count
+---
+
+# サークルサイズ
+
+円のサイズに関する基準は[ランキング基準](/wiki/Ranking_Criteria)を参照してください。
+
+**サークルサイズ** (***CS***)は[ヒットオブジェクト](/wiki/Hit_Object)のサイズを変更する[ビートマップ](/wiki/Beatmap)の難易度設定です。サークルサイズの範囲は0 ~ 10ですが、[ビートマップエディタ](/wiki/Client/Beatmap_editor)で指定できるのは2 ~ 7の範囲です。その他の値は[.osuファイル](/wiki/osu!_File_Formats/Osu_(file_format))を直接編集してください。
+
+## osu!
+
+osu!では、サークルサイズはサークルとスライダーのサイズを変更し、値を大きくするとヒットオブジェクトが小さくなります。スピナーはサークルサイズの影響を受けません。ヒットオブジェクトの大きさは次の式で計算できます。
+
+`r = 54.4 - 4.48 * CS`<!-- 古いリプレイのバグを解消するためには最後に 1.00041 を掛けます -->
+
+`r`は[osu!pixels](/wiki/osupixel)あたりのピクセル数を表し、`CS`はサークルサイズを表します。
```

Comment:
The original article doesn't have a link there, by the way.

Reply:
oh, i see, sorry.

Reply:
if you want to add it, you can do so by also adding the link to the original and other translations

Reply:
It would be better to add the link to others. The original is hard to know what osu!pixels is in detail.

Reply:
I'll do it when this pull been merge